lib/redis.js

```diff
@@ -28,6 +28,23 @@ function createClient(config, app) {
     client.on('error', function(error) {
       app.coreLogger.error(error);
     });
+  } else if (config.sentinels) {
+    assert(config.sentinels && config.sentinels.length !== 0, '[egg-redis] sentinels configuration is required when use redis sentinels');
+    assert(config.name, 'name is required in config along with sentinels');
+    config.sentinels.forEach(sentinel => {
+      assert(sentinel.host && sentinel.port,
+        '[egg-redis] host and port are required in config of sentinel node');
+      app.coreLogger.info('[egg-redis] connecting redis sentinel %s:%s', sentinel.host, sentinel.port);
+    });
+    client = new Redis(config);
+    client.on('connect', function() {
+      config.sentinels.forEach(sentinel => {
+        app.coreLogger.info('[egg-redis] connect success to sentinel %s:%s', sentinel.host, sentinel.port);
+      });
+    });
+    client.on('error', function(error) {
```

Comment:
Does the problem described here https://github.com/eggjs/egg-socket.io/pull/52#discussion_r230623112 still exist?

Reply:
it's ok now